wallet/addresses.go

```diff
@@ -565,6 +765,7 @@ func (w *Wallet) nextImportedXpubAddress(ctx context.Context, op errors.Op, mayb
 	addr = &xpubAddress{
 		AddressPubKeyHash: apkh,
 		xpub:              xpub,
+		account:           account,
```

Comment:
Seems to be missing `accountName` here.

On second thought, I see a bunch of "not used, can be faked" comments (but I imagine those are just because the addresses aren't propagated out of the wallet package) so unclear whether accountName should be here or not.

Reply:
nice catch

these are being returned by New{External,Internal}Address so they do propagate out of the package.